Sources/Host/Darwin/X86_64/MachX86_64.cpp

```diff
@@ -94,7 +94,7 @@ ErrorCode Mach::readCPUState(ProcessThreadId const &ptid,
 
 ErrorCode Mach::writeCPUState(ProcessThreadId const &ptid,
                               ProcessInfo const &pinfo,
-                              Architecture::CPUState const &state) {
+                              Architecture::CPUState const &ds2_state) {
```

Comment:
You should keep this named `state` and name your local variable `threadState`, or `context` or something like that.